__tests__/integration.test.js

```diff
@@ -0,0 +1,22 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const testData = require("../db/data/test-data/index.js");
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("/api/topics", () => {
+  test("GET:200 responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then((response) => {
```

Comment:
Nice testing, with good clear test descriptions. 
You might consider destructuring the body from the response, it can clean up your code a little. 
```
.then(({body}) => {
```
Take a look at our destructuring notes from Fundamentals if you're unsure of the syntax